lms/views/api/exceptions.py

```diff
@@ -250,12 +250,19 @@ def __json__(self, request):
             oauth2_token_service = request.find_service(name="oauth2_token")
 
             try:
-                oauth2_token_service.get()
+                kwargs = {}
+                if service := getattr(request.exception, "refresh_service", None):
```

Comment:
The use of `getattr` here follows the existing code above, but it might be clearer to test whether `request.exception` is an instance of `ExternalRequestError` instead.